app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/tracker_activity/DeviceShieldTrackerActivityViewModel.kt

```diff
@@ -131,6 +133,9 @@ class DeviceShieldTrackerActivityViewModel @Inject constructor(
     fun showAppTpEnabledCtaIfNeeded() {
         if (!vpnStore.didShowAppTpEnabledCta()) {
             vpnStore.appTpEnabledCtaDidShow()
+            val now = Instant.now().toEpochMilli()
+            val expiry = now.plus(TimeUnit.HOURS.toMillis(WINDOW_INTERVAL_HOURS))
+            vpnStore.setAppTPOnboardingBannerExpiryTimestamp(expiry)
```

Comment:
do we need to do the time conversion here? it could be done in the store and avoid that the `VM `knows about the internals.

Reply:
Thanks, updated :+1: